player/lua/osc.lua

```diff
@@ -1858,6 +1858,22 @@ layouts["topbar"] = function()
 end
 
 
+local function osc_visible(visible)
+    if state.osc_visible ~= visible then
+        state.osc_visible = visible
+        update_margins()
+    end
+    request_tick()
+end
+
+local function open_selector(type)
+    mp.command("script-binding select/select-" .. type)
+
+    if user_opts.visibility == "auto" then
+        osc_visible(false)
```

Comment:
This is merely an opinion that might match the use case for you guys:

On my OSC fork, I'm hiding the osc `osc_visible(false)` without checking `if user_opts.visibility == auto`

**Reason:**
Like we talked before, the OSC is at the bottom and `select/console` also renders at the bottom. I instantly got reports that OSC interferes (ie: hovering over area)

Thought I'd let you know, just in case. Since it would make sense to hide osc on `open_selector` regardless of visibility mode, because you want the focus to be on `select` once it is triggered without interference until ESC/leave `select`.

Reply:
Isn't that what the `osc/margins` user_property is for? To allow console/select to draw text above the OSC?

Reply:
Indeed it is, just thought I'd share it. I've opted for completely hiding osc while selector is on.

Like I said, merely an opinion that might match your use case.

Reply:
OSC with visibility always does not interfere because the console is rendered above it so I don't understand your point. If users intentionally make the OSC always visible why should we override their preference?

Reply:
👍 To each their own, just sharing an opinion